vscode/microsoft-kiota/package.json

```diff
@@ -542,7 +546,10 @@
     "@vscode/extension-telemetry": "^0.9.8",
     "@vscode/l10n": "^0.0.18",
     "adm-zip": "^0.5.16",
+    "appdata-path": "^1.0.0",
+    "follow-redirects": "^1.15.9",
     "is-online": "^11.0.0",
+    "jest": "^29.7.0",
```

Comment:
we should also remove the kiota runtime configuration below

Reply:
Yes. I can remove this as I am making the update to the scripts

Reply:
Done